### PR TITLE
Fix iPad Pro thumbnail size issues

### PR DESCRIPTION
**Pull request by orta**
There was an arbitrary max height, probably useful for the iPad at 1024x768, but less useful on the massive chunks of screens I have laying around on my desk.

### CODE REVIEW

**Comment by orta** [comment]
I checked rotation, seems all fine too FWIW

File: CHANGELOG.yml [hunk 1]
```diff
@@ -1,7 +1,18 @@
...
+      infrastructure:
+        - Fixes various memory leaks in live. - ash
+        - Fixes problems that arose from fixing memory leaks. - ash
+    user_facing:
+      - iPad Pro gets full screen lot thumbnails during Live Auctions - orta
 
+releases:
+  - version: 2.6.3
+    details: Bug fixes in Artist profile view
+    date: Not released yet
```
Comment (orta) [inline CHANGELOG.yml]: @alloy ^ if this release comes in the date will need adding

File: CHANGELOG.yml [hunk 2]
```diff
@@ -1,7 +1,18 @@
 upcoming:
-    version: 2.6.3
-    details: Bug fixes in Artist profile view
+    version: 2.6.4
+    details: Auctions Fixes?
+
+    dev:
+      infrastructure:
+        - Fixes various memory leaks in live. - ash
+        - Fixes problems that arose from fixing memory leaks. - ash
```
Comment (orta) [inline CHANGELOG.yml]: @ashfurrow these fixes weren't in 2.6.3, so I've moved them into a new reelease

**Comment by ashfurrow** [comment]
💯

File: CHANGELOG.yml (hunk 1 above)
Comment (alloy) [inline CHANGELOG.yml]: Okidoki 👍

**Pull request merged**